patches/chrome-browser-ui-webui-bookmarks-bookmarks_ui.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
-index 094a943ac95d0f159b223d3948e3b2e4b1759e76..f378bfe93dcaa890abb2221e5f3ffbfd45401fe0 100644
+index 094a943ac95d0f159b223d3948e3b2e4b1759e76..a2337eaf41f94aef033d1f2e9b38a255af35441e 100644
 --- a/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
 +++ b/chrome/browser/ui/webui/bookmarks/bookmarks_ui.cc
-@@ -46,6 +46,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
-   content::WebUIDataSource* source =
-       content::WebUIDataSource::Create(chrome::kChromeUIBookmarksHost);
+@@ -174,6 +174,7 @@ content::WebUIDataSource* CreateBookmarksUIHTMLSource(Profile* profile) {
+ 
+   source->UseStringsJs();
  
 +  BraveCustomizeBookmarksDataSource(source);
```

Comment:
Moved `BraveCustomizeBookmarksDataSource(source)` to the last of `CreateBookmarksUIHTMLSource()` to get a chance to replace already added strings.

Reply:
👍

Reply:
can you convert this to a `define` since you're changing it anyway?

Reply:
done.